.eslintrc.yaml

```diff
@@ -6,7 +6,7 @@ extends:
 env:
   node: true
   es6: true
-ignorePatterns: ["**/scripts", "**/__mocks__", "**/lib"]
+ignorePatterns: ["**/scripts", "**/__mocks__", "**/lib", "**/__tests__/__e2e__"]
```

Comment:
Is there a reason we don't want to lint check the e2e tests?? We check the rest of the test files